.readthedocs.yaml

```diff
@@ -21,4 +22,4 @@ build:
 sphinx:
   builder: html
   configuration: docs/source/conf.py
-  fail_on_warning: false
+  fail_on_warning: true
```

Comment:
Nit: The comment on Line 21 doesn't match the configuration here.

Reply:
thanks @ddl-joyce-zhao !